app/Http/Controllers/WikiController.php

```diff
@@ -110,12 +110,6 @@ public function create(Request $request): \Illuminate\Http\Response
               'wiki_id' => $wiki->id,
             ]);
 
-            // If we are local, the dev environment wont be able to run these jobs yet, so end this closure early.
-            // TODO maybe send different jobs instead? or do this in the jobs?
-            if (App::environment() === 'local') {
```

Comment:
Just so I understand: why is this not needed anymore?

Reply:
I'm not sure when this was needed in the first place, but nowadays the jobs that follow that check work. We just had the app env set to `production` everywhere prior to https://github.com/wmde/wbaas-deploy/commit/42a08af63af0d460d952938dce1520d76aa3c7e3#diff-bb11be128453a5ee0b102c8feaa9890809f7f925ac2dee60b620b1c351fdeb65R23, so the check wasn't even running